test/test_contraction_sequence.jl

```diff
@@ -23,7 +24,19 @@ ITensors.disable_warn_order()
   res_tree_sa = contract(tn; sequence=seq_tree_sa)[]
   seq_sa_bipartite = contraction_sequence(tn; alg="sa_bipartite")
   res_sa_bipartite = contract(tn; sequence=seq_sa_bipartite)[]
-  @test res_optimal ≈ res_greedy ≈ res_tree_sa ≈ res_sa_bipartite
+  seq_einexprs_exhaustive = contraction_sequence(tn; alg="einexpr", optimizer=Exhaustive())
+  res_einexprs_exhaustive = contract(tn; sequence=seq_einexprs_exhaustive)[]
+  seq_einexprs_greedy = contraction_sequence(tn; alg="einexpr", optimizer=Greedy())
+  res_einexprs_greedy = contract(tn; sequence=seq_einexprs_exhaustive)[]
+  seq_einexprs_kahypar = contraction_sequence(tn; alg="einexpr", optimizer=HyPar())
+  res_einexprs_kahypar = contract(tn; sequence=seq_einexprs_kahypar)[]
+  @test res_optimal ≈
+    res_greedy ≈
+    res_tree_sa ≈
+    res_sa_bipartite ≈
+    res_einexprs_exhaustive ≈
+    res_einexprs_greedy ≈
+    res_einexprs_kahypar
```

Comment:
I think it is better to split into separate tests, like:
```julia
  @test res_optimal ≈ res_greedy
  @test res_optimal ≈ res_tree_sa
  #...
```
The way it is written, if one failed the whole test would fail but we wouldn't know which one caused the failure.